test/fixtures/fail-npm-install/clean-package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Interestingly, different rules apply to `package.json` and `.json` files with a different name, which explains the tab discrepancy here. My guess is there's some sort of interplay between Prettier and `npm-package-json-lint`, but I'm not entirely sure either.